### PR TITLE
Add edit and delete buttons for orders

### PR DESCRIPTION
**Pull request by spaceman03**
This PR adds the edit and delete buttons for the orders page.

Ticket: https://github.com/orgs/ntuscse/projects/3?pane=issue&itemId=49448703

### CODE REVIEW

**Review by Iyzyman — DISMISSED** [review]

File: apps/cms/src/admin/views/MerchSales.tsx [hunk 1]
```diff
@@ -13,10 +13,18 @@ import { Table } from "payload/dist/admin/components/elements/Table";
...
+  const fetchData = async () => {
+    try {
+      const res: IOrder[] = await OrdersApi.getOrders();
+      setData(res);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
   useEffect(() => {
```
Comment (Iyzyman) [inline apps/cms/src/admin/views/MerchSales.tsx:26]: Try this instead

useEffect(() => {
  fetchData();
}, []);

File: apps/cms/src/admin/views/MerchSales.tsx [hunk 2]
```diff
@@ -13,10 +13,18 @@ import { Table } from "payload/dist/admin/components/elements/Table";
 const MerchSales: AdminView = ({ user, canAccessAdmin }) => {
   // Get data from API
   const [data, setData] = useState<IOrder[]>(null);
```
Comment (Iyzyman) [inline apps/cms/src/admin/views/MerchSales.tsx]: Try this instead

const [data, setData] = useState<IOrder[]>([]);

File: apps/cms/src/admin/views/MerchSales.tsx [hunk 3]
```diff
@@ -11,34 +11,43 @@ import SortedColumn from "../utils/SortedColumn";
 import { Table } from "payload/dist/admin/components/elements/Table";
 
 const MerchSales: AdminView = ({ user, canAccessAdmin }) => {
-
   // Get data from API
-  const [data, setData] = useState<IOrder[]>(null);
+  const [data, setData] = useState<IOrder[]>();
```
Comment (Iyzyman) [inline apps/cms/src/admin/views/MerchSales.tsx:15]: Try this instead
const [data, setData] = useState<IOrder[]>([]);

Comment (Iyzyman) [inline apps/cms/src/admin/views/MerchSales.tsx:15]: Can you change the code to the one below and try?

```
  const [data, setData] = useState<IOrder[]>(null);
  useEffect(() => {
    OrdersApi.getOrders()
      .then((res: IOrder[]) => setData(res))
      .catch((error) => console.log(error));
  }, []);
```

**Review by chanbakjsd — APPROVED** [review]
LGTM!

**Pull request merged**